tests/unit/test_extractive_summarization.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import nltk
```

Comment:
I think lines 4 and 6 are already [here](https://github.com/microsoft/nlp-recipes/blob/master/utils_nlp/dataset/__init__.py).